src/som/vmobjects/SArray.java

```diff
@@ -73,6 +75,23 @@ protected void copyIndexableFieldsTo(SArray destination) {
     }
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Do you still use this?

Reply:
If you are using Eclipse, I can recommend to run [UCDetector](http://www.ucdetector.org) on your code from time to time :)